src/views/Transcript/index.js

```diff
@@ -44,15 +49,20 @@ export default class Transcript extends Component {
     }
   }
 
-  //Compares each activity from TranscriptInfo to determine how to group by session
+  //Compares activity from activityList to previous activity' session to to determine how to group
+  //isUnique value is passed as a prop, along with Activity object, to TranscriptActivity Component
+  //Returns TranscriptActivity component which is passed into activityList
```

Comment:
Very helpful comment.  Thank you.

Maybe "...which should be passed into..." would be more accurate, since that's done outside this function.